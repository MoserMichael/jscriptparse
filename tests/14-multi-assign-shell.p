    out, status = system("echo 'hello world'")
    print("status: {status} output: {out}")
    
    a = "!hello"
    b = "world!"
    out, status = `echo "{a} {b}"`
    print("status: {status} output: {out}")

