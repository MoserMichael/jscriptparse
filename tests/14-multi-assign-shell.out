status: 0 output: hello world
status: 0 output: !hello world!
